baekjoon/fibonacci.js

```diff
@@ -0,0 +1,12 @@
+let input = require('fs').readFileSync('/dev/stdin').toString();
+const value = Number(input);
+
+function fibonacci(value) {
+    if (!value) return 0;
+    if (value === 1) return 1;
+
+    return fibonacci(value - 1) + ffibonacci(value - 2);
```

Comment:
ㅋㅋㅋ두 번째 fibonacci오타 났어요
```suggestion
    return fibonacci(value - 1) + fibonacci(value - 2);
```